pkg/agent/http_test.go

```diff
@@ -119,184 +216,22 @@ func Test_accessControl(t *testing.T) {
 
 	agt := mockAgent(t)
 	httpBackend := agt.httpBackend()
-
-	func() {
-		t.Log("federate testing begin ...")
-
-		tokenScenariosMap := map[string]map[string]samples.Scenario{
```

Comment:
the tokenScenario map was split into one entry per token in the `testCases` struct that is returned, this makes it much easier to execute each test case as a subtest, which produces much cleaner go test output (and allows us to filter by test cases as well, which is super useful).